contracts/modules/capital/Ramm.sol

```diff
@@ -122,7 +122,7 @@ contract Ramm is IRamm, MasterAwareV2 {
     nxmA = k / eth;
     nxmB = nxmB * eth / state.eth;
 
-    uint nxmOut = nxmA - state.nxmA;
+    uint nxmOut = state.nxmA - nxmA;
```

Comment:
I could be wrong, but it seems to me there is a bug here and it should be the other way around.

we expect new `nxmA` to be lower in **swapEthForNxm** as ETH goes in and NXM goes out.
```solidity
    eth = eth + ethIn; // new eth will be higher than state.eth (ETH in)
    nxmA = k / eth; // a higher eth divisor will mean a new lower nxmA (NXM out)
    nxmB = nxmB * eth / state.eth;

    // state.nxmA is higher (before NXM goes out)
    // new nxmA is lower (after NXM goes out)
    // therefore diff should be `state.nxmA - nxmA`
    uint nxmOut = nxmA - state.nxmA

```